crowdnotifier-sdk/src/main/proto/backend.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+package org.crowdnotifier.android.sdk.model;
+
+message QRCodeTrace {
+  uint32 version = 1;
+  MasterTrace masterTraceRecord = 2;
+}
+
+message MasterTrace {
+  bytes masterPublicKey = 1;
+  bytes masterSecretKeyLocation = 2;
+  bytes info = 3;
+  bytes nonce1 = 4;
+  bytes nonce2 = 5;
+  bytes cipherTextHealthAuthority = 6;
+}
+
+message PreTrace {
+  bytes identity = 1;
+  bytes partialSecretKeyForIdentityOfLocation = 2;
+  bytes cipherTextHealthAuthority = 3;
+  string message = 4;
```

Comment:
I don't think this should be set by the location, but rather by the health authority. So you'd normally not include it with PreTrace.

Reply:
This is only temporary. Our current version allows the VenueOwner to set a message. We will remove this in the future.

Reply:
ok, works for me :)